pkg/deployments/tasks/20230208-euler-linear-pool/readme.md

```diff
@@ -0,0 +1,9 @@
+# 2023-02-08 - Euler Linear Pool
+
+First deployment of the `EulerLinearPoolFactory`, for Linear Pools with an Euler yield-bearing token.
+
+## Useful Files
+
+- [Ethereum mainnet addresses](./output/mainnet.json)
+- [Goerli testnet addresses](./output/goerli.json)
+- [`EulerLinearPool` artifact](./artifact/EulerLinearPoolFactory.json)
```

Comment:
This also makes me realize that there are broken links in the READMEs for the recent Aave and ERC4626 deployments; the only artifact available is the factory itself, whereas Aave contains links for three artifacts and ERC4626 points only to a non-existent pool artifact.
```suggestion
- [`EulerLinearPoolFactory` artifact](./artifact/EulerLinearPoolFactory.json)
```

Reply:
Ok; will fix this (and the others). Good catch!